jobs/build/sweep/Jenkinsfile

```diff
@@ -75,34 +83,53 @@ node {
         return
     }
 
-    currentBuild.description = "Repairing state and sweeping new bugs.\n"
     def (major, minor) = commonlib.extractMajorMinorVersionNumbers(version)
-    def kinds = major >= 4 ? ["image", "extras"] : ["rpm"]
 
-    stage("Repair bug state") {
-        currentBuild.description += "* Moving attached bugs in MODIFIED state to ON_QA...\n"
-        for (kind in kinds) {
+    currentBuild.description = "Repairing state and sweeping new bugs.\n"
+
+    if (params.ATTACH_BUGS) {
+        stage("Attach bugs to advisories") {
```

Comment:
I don't like when stages vary between runs - it makes the stage view drop previous runs with mismatched stages. Could this whole stage be `Sweep bugs` and include both branches, and just update the description according to which branch is taken?